File: system-test/java/src/main/java/com/optivem/shop/dsl/core/usecase/external/tax/usecases/base/BaseTaxUseCase.java
package com.optivem.shop.dsl.core.usecase.external.tax.usecases.base;

import com.optivem.shop.dsl.core.shared.BaseUseCase;
import com.optivem.shop.dsl.core.shared.UseCaseContext;
import com.optivem.shop.dsl.driver.port.external.tax.TaxDriver;

public abstract class BaseTaxUseCase<TResponse, TVerification> extends BaseUseCase<TaxDriver, TResponse, TVerification> {
    protected BaseTaxUseCase(TaxDriver driver, UseCaseContext context) {
        super(driver, context);
    }
}

File: system-test/typescript/src/testkit/dsl/core/usecase/external/tax/usecases/base/BaseTaxUseCase.ts
import type { TaxDriver } from '../../../../../../../driver/port/external/tax/tax-driver.js';
import { BaseUseCase } from '../../../../../shared/base-use-case.js';

export abstract class BaseTaxUseCase<TResponse, TVerification> extends BaseUseCase<
  TaxDriver,
  TResponse,
  TVerification
> {}
